Makefile

```diff
@@ -33,4 +43,4 @@ show_environment:
 	@echo "Environment variables in use:"
 	@env | grep DM_ || true
 
-.PHONY: run_all run_app virtualenv requirements requirements_for_test frontend_build test test_pep8 test_python test_javascript show_environment
+.PHONY: run_all run_app virtualenv requirements requirements_for_test npm_build frontend_build test test_pep8 test_python test_javascript show_environment
```

Comment:
Sorry, my knowledge is not great about make files. Where is this `npm_build` defined? I see `npm_install`, although that is not listed in the `.PHONY`, how come?